File: site/swingjs/j2s/swingjs/plaf/JSCheckBoxMenuItemUI.js
Clazz.declarePackage ("swingjs.plaf");
Clazz.load (["swingjs.plaf.JSCheckBoxUI"], "swingjs.plaf.JSCheckBoxMenuItemUI", null, function () {
c$ = Clazz.declareType (swingjs.plaf, "JSCheckBoxMenuItemUI", swingjs.plaf.JSCheckBoxUI);
Clazz.overrideMethod (c$, "getDOMObject", 
function () {
return this.getButtonObject ("checkBox");
});
Clazz.overrideMethod (c$, "getPropertyPrefix", 
function () {
return "CheckBoxMenuItem.";
});
});

File: src/swingjs/plaf/JSCheckBoxMenuItemUI.java
package swingjs.plaf;

import jsjavax.swing.JMenuItem;

public class JSCheckBoxMenuItemUI extends JSCheckBoxUI {

	public JSCheckBoxMenuItemUI() {
		super();
		isMenuItem = true;
	}
	
	@Override
	protected String getPropertyPrefix() {
		return "CheckBoxMenuItem.";
	}

	@Override
	protected void installUIImpl() {
		menuItem = (JMenuItem) c;
		super.installUIImpl();
	}
}
